web/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -307,6 +310,11 @@ class Panel extends Component<PanelProps, PanelState> {
             <Col>{warning && <Alert color="warning">{warning}</Alert>}</Col>
           </Row>
         ))}
+        {this.state.infos?.map((info, index) => (
+          <Row key={index}>
+            <Col>{info && <Alert color="success">{info}</Alert>}</Col>
```

Comment:
`success` is green, I'd pick `info` as the info-level color (blue/turquoise-ish):

```suggestion
            <Col>{info && <Alert color="info">{info}</Alert>}</Col>
```